### PR TITLE
Fixed GUI misaligning the panels on large displays

### PR DESCRIPTION
**Pull request by ironicnet**
Deleted the tableLayoutPanel1 and put the PanelSource and PanelDestination directly in the form.

Modified the anchor to Top,Left always

### CODE REVIEW

**Comment by igitur** [comment]
Thanks. I'll check it out. Don't merge yet.

**Comment by ironicnet** [comment]
No problem. I will stop auto merging my PRs, until someone else gives an approval.

File: OpenDBDiff/Front/PrincipalForm.Designer.cs [hunk 1]
```diff
@@ -208,28 +206,28 @@ private void InitializeComponent()
             this.txtOldObject.Styles.LineNumber.Size = 9F;
             this.txtOldObject.Styles.Max.Size = 9F;
             this.txtOldObject.TabIndex = 0;
-            //
+            //
```
Comment (igitur) [inline OpenDBDiff/Front/PrincipalForm.Designer.cs]: Please undo all the whitespace changes.

File: OpenDBDiff/Front/PrincipalForm.Designer.cs [hunk 2]
```diff
@@ -658,9 +623,24 @@ private void InitializeComponent()
             this.PanelActions.Name = "PanelActions";
             this.PanelActions.Size = new System.Drawing.Size(98, 483);
             this.PanelActions.TabIndex = 17;
-            //
+            //
```
Comment (igitur) [inline OpenDBDiff/Front/PrincipalForm.Designer.cs]: Why did you move this block?

File: OpenDBDiff/Front/PrincipalForm.Designer.cs (hunk 1 above)
Comment (jspraul) [reply OpenDBDiff/Front/PrincipalForm.Designer.cs]: Is this something covered by .editorconfig or CodeMaid? If this work was done before those, I vote to just let it ride in the .designer.cs files -- do a one-time whitespace cleanup pull request and then lean on .editorconfig/CodeMaid moving forward.

File: OpenDBDiff/Front/PrincipalForm.Designer.cs (hunk 2 above)
Comment (jspraul) [reply OpenDBDiff/Front/PrincipalForm.Designer.cs]: Probably just a Visual Studio .designer.cs randomness due to changes in control order... I vote for letting people stomp all over order in the .designer.cs files as long as nothing breaks.

Comment (igitur) [reply OpenDBDiff/Front/PrincipalForm.Designer.cs]: In that case, when 2 people are working on different branches you're bound to get conflicts, or at least a corrupt designer file.

Also, as I've just experienced, extremely difficult to identify the actual logical change and revert it.

Comment (jspraul) [reply OpenDBDiff/Front/PrincipalForm.Designer.cs]: Fair enough. It sounds like the solution is "WPF"! Manually cleaning up auto-generated order changes is decade+ struggle that discourages fixes to the current UI, but I don't see anything that can help with it.

I appreciate you taking the time to look things over. It sounds like this will be tedious for at least one person (contributor or reviewer) and in this case seems to be both. Keep up the good work!

https://www.google.com/search?q=.net+winforms+designer+source+control+conflict

http://blog.plasticscm.com/2013/11/the-sortresx-tool.html

**Comment by igitur** [comment]
Can you rebase this on the `develop` branch, please?

**Comment by igitur** [comment]
I release a new version (0.12.0) that slightly redesigns the main form. If there are still other changes you want to make, please let me know. Closing this for now.
